src/parser/error-parser.js

```diff
@@ -0,0 +1,15 @@
+const se = require('../utils/safe-extract');
+const ERROR_MESSAGE = 'error_message',
+  HEADERS = 'headers',
+  JSON_STRING = 'json';
+
```

Comment:
I did this method to tentatively parse the error message before returning it too. However i am not sure about the nature of the different messages so I dont know if it would eliminate relevant info for the calc end. So far I only seen two errors: 

* When the time is in the past, which logs an error message.
* When an address is invalid in which case no error message is logged and one of the geocode status will display "ZERO_RESULTS".